### ISSUE TITLE
RStudio 1.2 Constant GPU Dependency

### ISSUE THREAD

**Issue by joelmlevin** [rstudio/rstudio#4672]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.2.1335
    OS Version      : OSX Mojave 10.14.2
    R Version       : 3.5.1 (2018-07-02)
    Machine: MacBook Pro (15-inch, 2017); Radeon Pro 555 Graphics

### Steps to reproduce the problem

Open RStudio.

### Describe the problem in detail

RStudio causes OSX to use discrete (rather than integrated) graphics whenever it is open. This results in significantly increased battery use, even when RStudio is not actively doing anything.

### Describe the behavior you expected

With previous versions of RStudio, this did not happen.

**Comment by kevinushey** [comment]
We do have a fix for this in the daily builds:

https://github.com/rstudio/rstudio/commit/bec49089ea9f1fcf9210e5d5aa32414e63caa56e#diff-00d5d8340bdd93db554d426ab7fd944e

but we're a bit wary of backporting this change as we've already had so many rendering-related headaches through the v1.2 release cycle and don't want to risk new regressions in the patch release.

**Comment by kevinushey** [comment]
Marking as v1.2-patch since this has been reported by other users.

**Comment by kevinushey** [comment]
Patch ported to v1.2-patch; it will become part of the next [preview release](https://www.rstudio.com/products/rstudio/download/preview/).

**Comment by ronblum** [comment]
@joelmlevin Would you be willing to check [the latest preview release](https://www.rstudio.com/products/rstudio/download/preview/) to see if the problem is resolved?

**Comment by joelmlevin** [comment]
> @joelmlevin Would you be willing to check [the latest preview release](https://www.rstudio.com/products/rstudio/download/preview/) to see if the problem is resolved?

I am currently running Version 1.3.149, which I believe I downloaded as a daily release per @kevinushey 's suggestion. I am not currently experiencing the issue. Sorry that I did not update this thread with that information.

I'm a bit hesitant to change anything, given that the current version is working fine for me right now, but I can if it would be very helpful.
